playground/mongo/Mongo.AppHost/Properties/launchSettings.json

```diff
@@ -5,12 +5,12 @@
       "commandName": "Project",
       "dotnetRunMessages": true,
       "launchBrowser": true,
-      "applicationUrl": "http://localhost:15888",
+      "applicationUrl": "http://localhost:15888/",
```

Comment:
Why this change?

Reply:
I'll revert. Was doing some experimentation with a dotnet watch "bug" / "feature"

Reply:
Was just experimenting with some behaviors in `dotnet watch`.